Umweltzone/src/main/res/values-es/strings.xml

```diff
@@ -0,0 +1,375 @@
+        listo para ser enviado.]]>
+    </string>
+    <string name="zone_not_drawable_later">Tal vez después</string>
+    <string name="zone_not_drawable_open_email">Abrir email</string>
+    <string name="zone_not_drawable_no_email_app">
+        No se ha encontrado una aplicación de correo electrónico compatible.
+    </string>
+    <string name="zone_not_drawable_app_chooser_title">Abrir borrador con &#8230;</string>
+    <string name="zone_not_drawable_email_subject">
+        Umweltzone Android App / Integración <xliff:g example="Wiesbaden" id="zoneName">%1$s</xliff:g>
+    </string>
+
+    <string name="zone_not_drawable_email_html_message">
+        <![CDATA[<p>Estimado Señor o Señora.<br />
+        <br />
+        Estoy usando la aplicación gratuita y de código libre &quot;Umweltzone&quot; [1]
+        que me permite estár al tanto de la ubicación de las zonas de baja emisión y
+        sobre las regulaciones particulares en varias ciudades.
+        Considero que esta aplicación como un complemento útil a las fuentes de información que se
+        ofrecen sobre las ciudades.<br />
+        <br />
+        Desfortunadamente, la <b>zona de baja emisión de %1$s</b> no puede ser mostrada.
+        Debido a que las coordenadas geográficas que describen la ubicación de la misma no
+        ha sido publicadas.
+        Por lo tanto, me gustaría preguntar personalmente para liberar las coordenadas geográficas
+        bajo un <b> licencia libre </ b>.
+        Para facilitar el procesamientos de la información, es idóneo que los datos se encuentren
+        en alguno de los siguientes formatos <b>legibles para una máquina</b>:
+        ESRI Shapefiles, CSV, GeoJSON.<br />
+        <br />
+        Atentamente</p>
+        <p>----<br />
+        <br />
+        Sehr geehrte Damen und Herren,<br />
+        <br />
+        ich nutze die kostenfreie und quelloffene Anwendung &quot;Umweltzone&quot; [1],
+        mit der ich mich über den Verlauf von Umweltzonen und individuelle
+        Regelungen in verschiedenen Städten auf meinem Android-Gerät
+        informieren kann. Ich betrachte diese Anwendung als nützliche
+        Erweiterung zum bestehenden Informationsangebot vieler Städte.<br />
+        <br />
+        Leider kann die <b>Umweltzone %1$s</b> nicht dargestellt werden. Es fehlen
+        die Geokoordinaten des Umweltzonenumrings. Daher möchte ich Sie hiermit
+        persönlich bitten, die Koordinaten unter einer <b>freien Lizenz</b> zu
+        veröffentlichen. Um eine Weiterverarbeitung zu ermöglichen, eignen
+        sich folgenden <b>maschinenlesbare Dateiformate</b>:
+        ESRI Shapefiles, CSV, GeoJSON.<br />
+        <br />
+        Mit freundlichen Grüßen</p>
+
+        <p>___<br />
+        [1]: <a href="https://play.google.com/store/apps/details?id=de.avpptr.umweltzone">https://play.google.com/store/apps/details?id=de.avpptr.umweltzone</a></p>]]>
+    </string>
+
+    <!-- TraceDroid -->
```

Comment:
Please also add a string for `trace_droid_button_title_no`. It might be the same word as in English but that's fine.

Reply:
to close of no translatable xD